casper/src/main/scala/coop/rchain/casper/util/rholang/InterpreterUtil.scala

```diff
@@ -89,59 +92,61 @@ object InterpreterUtil {
         }
     }
 
-  private def processPreStateHash[F[_]: Monad: Log: BlockStore](
+  private def processPreStateHash[F[_]: Sync: Log: BlockStore: TaskLift](
```

Comment:
Let's also keep generic typeclass constraints before/after the project specific ones. So here it'd be `[F[_]: Sync: TaskLift: Log: BlockStore]` or `[F[_]: Log: BlockStore: Sync: TaskLift]` depending on what's more common in the project. Totally optional for this PR though

Reply:
Makes sense